docs/security/authorization-pushdown.md

```diff
@@ -28,6 +28,27 @@ In order to use the Authorization Ranger Hive Plugin, you need to configure the
 
 Once you have used the correct configuration, you can perform authorization operations by calling Gravitino [authorization RESTful API](https://gravitino.apache.org/docs/latest/api/rest/grant-roles-to-a-user).
 
+#### Gravitino authorization privilege mapping Hive privilege property configurations
+
+If you enable these property configurations, The RangerAuthorizationHivePlugin will use these properties to override the default configuration in the program.
+
+| Property Name                                             | Description                                                                           | Default Value        | Since Version |
+|-----------------------------------------------------------|---------------------------------------------------------------------------------------|----------------------|---------------|
+| `gravitino.authorization.owner.privileges`                | The owner privileges, the owner can do anything on the metadata object configuration. | `all`                | 0.6.1         |
+| `gravitino.authorization.privilege.mapping.CREATE_SCHEMA` | Gravitino `CREATE_SCHEMA` privilege mapping Ranger privileges.                        | `create`             | 0.6.1         |
+| `gravitino.authorization.privilege.mapping.CREATE_TABLE`  | Gravitino `CREATE_TABLE` privilege mapping Ranger privileges.                         | `create`             | 0.6.1         |
+| `gravitino.authorization.privilege.mapping.SELECT_TABLE`  | Gravitino `SELECT_TABLE` privilege mapping Ranger privileges.                         | `read,select`        | 0.6.1         |
+| `gravitino.authorization.privilege.mapping.MODIFY_TABLE`  | Gravitino `MODIFY_TABLE` privilege mapping Ranger privileges.                         | `update,alter,write` | 0.6.1         |
+
+:::caution
+Ranger Hive privileges enum RangerHivePrivilege: create, alter, drop, index, lock, select, insert, update, delete, read, write, all
+defined in the `authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerPrivilege.java`
+Format: gravitino.authorization.owner.privileges = <RANGER-PRIVILEGE>,<RANGER-PRIVILEGE>,...
+
+Gravitino privileges defined in the `api/src/main/java/org/apache/gravitino/authorization/Privilege.java`
+Format: authorization.privilege.mapping.<GRAVITINO-PRIVILEGE> = <RANGER-PRIVILEGE>,<RANGER-PRIVILEGE>,...
+:::
```

Comment:
I think the document here is not enough. You should clearly describe how to configure, what is the meaning, and give an example here.

Also, do we only support 4 privileges for mapping, how do we support more privileges to map easily?

Reply:
> I think the document here is not enough. You should clearly describe how to configure, what is the meaning, and give an example here.

OK, I added example in the docs.

> Also, do we only support 4 privileges for mapping, how do we support more privileges to map easily?

No, We can support all Gravitino privileges to configurations.
Because I use 

```
Privilege.Name gravitinoPrivilege =
                    Privilege.Name.valueOf(key.trim().toUpperCase());
```
These codes can convert any String to a Gravitino Privilege enum value.